README.md

```diff
@@ -213,6 +213,17 @@ If you want to get involved with helping us grow the CLI, whether that is sugges
 
 Checkout our [Wiki](https://github.com/pnp/cli-microsoft365/wiki) for guides on how to contribute to this project.
 
+## Need Help?
+
+<h4 align="center">
+  Join our community
+</h4>
+<p align="center">
+  <a href="https://discord.com/invite/7rfW4kg6B5">
```

Comment:
Let's use our new link: https://discord.gg/35HTFJ544b

Reply:
Shouldn't we wait and use the aka link?

Reply:
https://github.com/pnp/cli-microsoft365/pull/4635#issuecomment-1463807833

Gary said not to wait for it.

Reply:
I think there was a bit different case as we were fixing a bug which we wanted to do ASAP.
Here we are adding a totally new section not yet present in readme. I would say it would be better to do it once and not open a another PR in a bit 😉.
But ok, today I will update the link during coding (night) time 👍

Reply:
done

Reply:
> I think there was a bit different case as we were fixing a bug which we wanted to do ASAP.
> Here we are adding a totally new section not yet present in readme. I would say it would be better to do it once and not open a another PR in a bit 😉.
> But ok, today I will update the link during coding (night) time 👍

I guess it doesn't really matter since we'll have to update all links to the aka.ms link anyway in the near future.

Reply:
True